django/db/models/fields/related_descriptors.py

```diff
@@ -195,6 +195,13 @@ def get_prefetch_querysets(self, instances, querysets=None):
         else:
             query = {"%s__in" % self.field.related_query_name(): instances}
         queryset = queryset.filter(**query)
+        # Since there can be only one object prefetched for each instance
+        # order by is insignificant and slows down the query.
+        # queryset = queryset.order_by()
+        # queryset was cloned by filter() just above, no need to re-clone it.
+        queryset.query.clear_ordering()
+        # No need to handle the case where the query is_sliced (LIMIT)
+        # because no filter can be added to a sliced query.
```

Comment:
Again, we want to `force=False` as users might compose a complex query through `Prefetch(queryset)` where distinct and / or slicing requires `ORDER BY` to remain present to preserve the semantic of the query.

```suggestion
        # There can be only one object prefetched for each instance so
        # clear ordering if the query allows it without side effects.
        queryset.query.clear_ordering(force=False)
```

There is no need to be overly aggressive in the optimization given `clear_ordering(force=False)` will handle
1. `Meta.ordering` which is the reported issue in this ticket
2. An explicitly provided `order_by` assuming neither `distinct` or `slicing` is specified (which can be handy when windows functions are involved)

Reply:
`def clear_ordering(self, force=False, clear_default=True):`
`force=False` is the default.
I don't understand your reasoning if you want to explicit `force=False` and not `clear_default=True`.